.github/workflows/validate.yml

```diff
@@ -30,7 +30,7 @@ jobs:
       fail-fast: false
       matrix:
         node: [14, 16, 18]
-        react: [latest, canary, experimental]
+        react: ['18.x', latest, canary, experimental]
```

Comment:
Added now so that we automatically continue testing React 18.x once React 19 is released.